plugins/auth-backend/src/lib/OAuthProvider.ts

```diff
@@ -41,8 +41,12 @@ export type Options = {
 };
 
 const readState = (stateString: string): OAuthState => {
+
+  // This is hardcoded for Azure (breaks other providers) until workaround is found
+  const fixedState = stateString.substring(38)
```

Comment:
This is the hardcoded workaround I've put in to get the Azure provider functional.